.github/workflows/release.yml

```diff
@@ -46,3 +46,12 @@ jobs:
         run: |
           make buildx.image
           make buildx.push
+
+  trivy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Scan
+      uses: docker://aquasec/trivy
+      with:
+        args: --severity CRITICAL --no-progress -t <release-tag>
```

Comment:
`<release-tags>` Does this argument work.  I think this is the placeholder, which you need to replace with the actual release tag.